src/ra.erl

```diff
@@ -716,6 +719,13 @@ new_uid(Source) when is_binary(Source) ->
     Prefix = ra_lib:derive_safe_string(Source, 6),
     ra_lib:make_uid(string:uppercase(Prefix)).
 
+%% @doc generates a random uid using timestamp for the first
+%% 6 characters.
+%% @end
+new_nvid() ->
```

Comment:
Do we need a prefix at all?

Reply:
It is there mostly to compensate for non-crypto source in the random part.

But related: I've been on and off the idea of re-using UID for non-voter identity. I chose a separate ID for clarity, but there is appeal in re-using what we already have. What do you think?

Reply:
I don't think a node's identity changes when its status become a non-voter, so I'd say the same ID can be used?